From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Make put_device() warn that it will sleep

This makes scsi emit messages all over the place...
But it's something that we need to fix up.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/base/core.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/base/core.c
+++ gregkh-2.6/drivers/base/core.c
@@ -557,6 +557,7 @@ struct device * get_device(struct device
  */
 void put_device(struct device * dev)
 {
+	/* might_sleep(); */
 	if (dev)
 		kobject_put(&dev->kobj);
 }
